src/Keboola/StorageApi/Client.php

```diff
@@ -2377,8 +2377,10 @@ public function apiGet($url, $fileName = null, $requestOptions = [])
     public function apiPost($url, $postData = null, $handleAsyncTask = true, $requestOptions = [])
     {
         $requestOptions = $this->filterRequestOptions($requestOptions);
-        $requestOptions['form_params'] = $postData;
-        return $this->request('post', $url, $requestOptions, null, $handleAsyncTask);
+        if ($postData !== null && $postData !== []) {
+            $requestOptions['form_params'] = $postData;
+        }
```

Comment:
Proč tohle? A proč do té formdata metody? Není bezpečnější ji nechat jak je?

Reply:
No, proto jsem se predtim ptal, jestli to muze neco rozbit :-)
Prijde mi lepsi, ze se data poslou, jen kdyz jsou fakt vyplneny a ne defaultni prazdny hodnoty.

Reply:
Ale OK, u tech form-data callu to vratim zpatky...